<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalRecordEditView</class>
 <widget class="QWidget" name="MedicalRecordEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>就诊记录编辑</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* 设置背景颜色和字体样式 */
    QWidget {
        background-color: #f5f5f5;
        font-family: Arial, sans-serif;
        font-size: 12px;
    }

    /* 设置 QLabel 的样式 */
    QLabel {
        color: #333333;
        font-weight: bold;
    }

    /* 设置 QLineEdit 的样式 */
    QLineEdit {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 4px;
        background: #ffffff;
    }
 QPlainTextEdit {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 4px;
        background: #ffffff;
    }

    /* 设置 QComboBox 的样式 */
    QComboBox {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 4px;
        background: #ffffff;
    }

    /* 设置 QSpinBox 的样式 */
    QSpinBox {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 4px;
        background: #ffffff;
    }

    /* 设置 QPushButton 的样式 */
    QPushButton {
        background-color: #4CAF50;
        color: white;
        border: none;
        border-radius: 4px;
        padding: 8px 16px;
        font-size: 14px;
    }

    QPushButton:hover {
        background-color: #45a049;
    }

    QPushButton#button_cancel {
        background-color: #f44336;
    }

    QPushButton#button_cancel:hover {
        background-color: #d32f2f;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>编号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="dbEditID"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>患者姓名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dbEditPatientName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>主治医生：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dbEditDoctorName"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>诊断结果：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>开具药品：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>就诊日期：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="dbEditDate">
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="dbEditResult"/>
     </item>
     <item row="4" column="1">
      <widget class="QPlainTextEdit" name="dbEditDrug"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/cancel.png</normaloff>:/img/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btSave">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="icon">
        <iconset resource="lab4.qrc">
         <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lab4.qrc"/>
 </resources>
 <connections/>
</ui>
